### DISCUSSION TITLE
WDIO v9 - webSocketUrl setting in conf irrelevant?

### DISCUSSION THREAD

**Discussion by th0car** [webdriverio/webdriverio#13846] [Technical Discussion]
Is the webSocketUrl value in wdio.conf irrelevant in WDIO v9? 
(In my webdriver logs this value is always true when the session is created, no matter what value I set in the conf)

**Comment by christian-bromann** [comment]
> Is the webSocketUrl value in wdio.conf irrelevant in WDIO v9

Yes, as mentioned in our [release blog post](https://webdriver.io/blog/2024/08/15/webdriverio-v9-release) we now automatically enable WebDriver Bidi except you set `wdio:enforceWebDriverClassic` to opt out of it.
